realsense2_camera/launch/includes/nodelet.launch.xml

```diff
@@ -87,13 +87,17 @@
   <arg name="aligned_depth_to_fisheye1_frame_id" default="$(arg tf_prefix)_aligned_depth_to_fisheye1_frame"/>
   <arg name="aligned_depth_to_fisheye2_frame_id" default="$(arg tf_prefix)_aligned_depth_to_fisheye2_frame"/>
 
-  <arg name="publish_tf"               default="true"/>
-  <arg name="tf_publish_rate"          default="0"/> <!-- 0 - static transform -->
+  <arg name="publish_tf"                default="true"/>
+  <arg name="tf_publish_rate"           default="0"/> <!-- 0 - static transform -->
 
-  <arg name="odom_frame_id"            default="$(arg tf_prefix)_odom_frame"/>
-  <arg name="topic_odom_in"            default="$(arg tf_prefix)/odom_in"/>
-  <arg name="calib_odom_file"          default=""/>
-  <arg name="publish_odom_tf"          default="true"/>
+  <arg name="odom_frame_id"             default="$(arg tf_prefix)_odom_frame"/>
+  <arg name="topic_odom_in"             default="$(arg tf_prefix)/odom_in"/>
+  <arg name="calib_odom_file"           default=""/>
+  <arg name="publish_odom_tf"           default="true"/>
```

Comment:
not sure what github caught as changes in here, as it seems like nothing has changed